README.md

```diff
@@ -85,7 +86,6 @@ All of mommy's NSFW content is hidden behind CARGO_MOMMYS_MOODS, where "thirsty"
 
 You can enable "true mommy chaos mode" by setting `CARGO_MOMMYS_MOODS="chill/thirsty/yikes"`, making mommy oscillate wildly between light positive affirmation and trying to break you in half~
 
-* `CARGO_MOMMYS_MOODS` - how kinky mommy will be~ (default: "chill", possible values "chill", "thirsty", "yikes")

```

Comment:
Uhh this isn't quite right, the fully list should still be available.

Reply:
Not sure if this is what you meant?